src/biokbase/narrative/system.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
This is the bulk of the change - migrate `system_variable` and `strict_system_variable` over here.

Really, these should be separate functions - `get_workspace`, `get_workspace_id`, etc. But that's out of scope for this little project. It was originally done with strings to map directly to various app spec requests.